### ISSUE TITLE
malltina.com not loading

### ISSUE THREAD

**Issue by mkoushan** [ThomazPom/Moz-Ext-UltimaDark#98]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I am on a compatible (Desktop) browser that is up-to-date or explicitly listed as being compatible in the [README](https://github.com/ThomazPom/Moz-Ext-UltimaDark/blob/master/readme.md)
- [X] I'm reporting that UltimaDark is broken in a novel way that deserves it's own issue meaning it should not be added to [#6](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues/6) [#7](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues/7) [#8](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues/8) or [#94](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues/94)
- [X] This site is not listed in an existing issue or in [#6](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues/6) [#7](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues/7) [#8](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues/8) or [#94](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues/94)
- [X] I verified that I am on the latest version
- [X] I verified that ImageEdition, Caching, Service Worker Compatibility and the extension itself are all enabled and that I loaded the site cleanly by closing the tab and opening the URL in a new tab or I flushed it completely with [Tab Suspender](https://github.com/Hau-Hau/firefox-tab-suspender)
- [X] I tried disabling caching and refreshing the page several times and it is still broken.
- [X] I confirmed that the target site is not on the excluded sites list or a Mozilla domain on Firefox's protected site listin about:config
- [X] I've searched [known issues](https://github.com/ThomazPom/Moz-Ext-UltimaDark/issues) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [README](https://github.com/ThomazPom/Moz-Ext-UltimaDark/blob/master/readme.md) and understand how and why to open an issue.

### Site URL

https://malltina.com/

### Provide a description of what about the site is broken and attach screenshots at the bottom

The whole site won't load when the extension is enabled. After disabling the extension it'll load normally.

** Enabled: **
![image](https://github.com/user-attachments/assets/87718ed8-c7e2-4174-8930-511efecc8dc4)

** Disabled: **
![image](https://github.com/user-attachments/assets/79ba9126-436a-45c6-a7b5-bc7f995c5ef2)

### Provide console logs

- [X] Go to about:debugging#/runtime/this-firefox
- [X] Click "Inspect" on UltimaDark
- [X] Clear the log and refresh the page that is broken
- [X] Copy the WHOLE output and insert it below

### Complete Log

```shell
Creating internal property for 
function Element()
 innerHTML background.js:12:21
Creating internal property for 
function ShadowRoot()
 innerHTML background.js:12:21
Creating internal property for 
function CSS2Properties()
 backgroundColor background.js:12:21
Creating internal property for 
function Navigator()
 serviceWorker background.js:12:21
UltimaDark Content script install 
Window https://malltina.com/
background.js:2034:15
Creating internal property for 
function Element()
 innerHTML eval:9:21
Creating internal property for 
function ShadowRoot()
 innerHTML eval:9:21
Creating internal property for 
function CSS2Properties()
 backgroundColor eval:9:21
Creating internal property for 
function Navigator()
 serviceWorker eval:9:21
UltimaDark Content script override website 
Window https://malltina.com/
eval:11:15
UltimaDark Websites overrides install 
Window https://malltina.com/
eval:47:15
Editing function : 
function HTMLObjectElement()
 
function checkValidity()
eval:316:19
Editing function : 
function CSSStyleDeclaration()
 
function setProperty()
eval:316:19
Editing function : 
function CSSStyleSheet()
 
function replace()
eval:316:19
Editing function : 
function CSSStyleSheet()
 
function replaceSync()
eval:316:19
Editing property : 
Array [ Element(), ShadowRoot() ]
 innerHTML eval:255:19
Editing property : 
function Element()
 innerHTML eval:255:19
Editing property : 
function ShadowRoot()
 innerHTML eval:255:19
Editing property : 
Array [ HTMLIFrameElement(), HTMLEmbedElement() ]
 src eval:255:19
Editing property : 
function HTMLIFrameElement()
 src eval:255:19
Editing property : 
function HTMLEmbedElement()
 src eval:255:19
Editing property : 
Array [ HTMLObjectElement() ]
 data eval:255:19
Editing property : 
function HTMLObjectElement()
 data eval:255:19
Editing property : 
Array [ HTMLIFrameElement() ]
 srcdoc eval:255:19
Editing property : 
function HTMLIFrameElement()
 srcdoc eval:255:19
Editing property : 
function Element()
 outerHTML eval:255:19
Editing function : 
function Element()
 
function insertAdjacentHTML()
eval:316:19
Editing function : 
function Element()
 
function setAttribute()
eval:316:19
Editing property : 
function HTMLImageElement()
 src eval:255:19
Editing property : 
function SVGImageElement()
 href eval:255:19
Editing property : 
function HTMLLinkElement()
 href eval:255:19
Wrapping function : 
function SVGSVGElement()
 
function value()
 setAttribute eval:294:19
Wrapping function : 
function HTMLUnknownElement()
 
function value()
 setAttribute eval:294:19
Editing function : 
function Node()
 
function appendChild()
eval:316:19
Editing function : 
function Node()
 
function insertBefore()
eval:316:19
Editing property : 
function Node()
 textContent eval:255:19
Editing property : 
function CSS2Properties()
 background eval:255:19
Editing property : 
function CSS2Properties()
 fill eval:255:19
Editing property : 
function CSSStyleDeclaration()
 cssText eval:255:19
Editing function : 
function CSSStyleSheet()
 
function addRule()
eval:316:19
Editing function : 
function CSSStyleSheet()
 
function insertRule()
eval:316:19
Editing property : 
function CSS2Properties()
 backgroundColor eval:255:19
Editing property : 
function CSS2Properties()
 background-color eval:255:19
Editing property : 
function CSS2Properties()
 color eval:255:19
Editing property : 
Array [ HTMLElement(), SVGElement() ]
 style eval:255:19
Editing property : 
function HTMLElement()
 style eval:255:19
Editing property : 
function SVGElement()
 style eval:255:19
Editing property : 
function HTMLElement()
 innerText eval:255:19
UltimaDark Websites overrides ready 
Window https://malltina.com/
 elapsed: 2 eval:312:15
Resolve IDK Vars is ready background.js:2134:15
UltimaDark Content script ready 
Window https://malltina.com/
background.js:2142:15
The resource at “https://malltina.com/_next/static/chunks/webpack-5131e486dd9a4d5d.js” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly. malltina.com
The resource at “https://malltina.com/_next/static/css/b66ed6fa7a2dfced.css” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly. malltina.com
The resource at “https://malltina.com/_next/static/css/319bde88aac62ba5.css” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly. malltina.com
The resource at “https://malltina.com/_next/static/css/fe8b1ef7aa785882.css” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly. malltina.com
```

**Comment by ThomazPom** [comment]
Thank you for reporting this interesting bug.

**Comment by mkoushan** [comment]
same issue with [this](https://alternativeto.net/)

**Comment by ThomazPom** [comment]
At this point, me and this bug are kind of old friends. However i home never see it again, I've fixed it in https://github.com/ThomazPom/Moz-Ext-UltimaDark/commit/ccc8195faf339a0fdac81f389dcb88c37530579d, and will be available in the 1.5.66 i'm releasing now.

**Comment by mkoushan** [comment]
Thanks, https://github.com/ThomazPom/Moz-Ext-UltimaDark/commit/ccc8195faf339a0fdac81f389dcb88c37530579d solved the issue (btw I found another issue I'll create a new one for it)
